src/app/models/form.server.model.ts

```diff
@@ -153,10 +153,6 @@ const EncryptedFormSchema = new Schema<IEncryptedFormSchema>({
   },
 
   payments_field: {
```

Comment:
Now that `enabled` is no longer in `payments_field` and that `payments_field` is strongly tied to the existence of the field itself. We should actually make the field items `description` & `amount_cents` required as per the interaction in FE.

Reply:
No eh we can't, because then what will the values be before payment account connection?

Reply:
Oh I mean `payments_field` itself being optional but `payments_field.description` and `payments_field.amount_cents` being required. Not sure if thats possible but previously @justynoh and I thought that was the cleaner way to go. So upon stripe connection -> `payments_field` still doesn't exist

Reply:
Also, we don't want to display payments before they get a chance to edit the description and amount.

Reply:
Wait can meh HAHA

Reply:
Will push this off to another PR bah